TeachingAssistantProfesor/app/src/main/java/malvolyo/teachingassistantprofesor/DisplayCourseQuestionsActivity.java

```diff
@@ -0,0 +1,162 @@
+package malvolyo.teachingassistantprofesor;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Display;
+import android.view.View;
+import android.widget.Button;
+import android.widget.RelativeLayout;
+import android.widget.Toast;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+
+public class DisplayCourseQuestionsActivity extends AppCompatActivity {
+    private DatabaseReference database;
+    private String today;
+    private RelativeLayout questions;
+    private RelativeLayout rl;
+    private String course;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_display_course_questions);
+
+        Intent intent = getIntent();
+        course = intent.getStringExtra(Constants.COURSE);
+
+        questions = new RelativeLayout(this);
+        rl = (RelativeLayout) findViewById(R.id.activity_display_course_questions);
+
+        database = FirebaseDatabase.getInstance().getReference();
+        setDay();
+
+        database.child(course).child(DBConstants.COURSE_QUESTIONS).child(today).
+                child(DBConstants.QUESTIONS).
+                addValueEventListener(new ValueEventListener() {
+                    @Override
+                    public void onDataChange(DataSnapshot dataSnapshot) {
+                        Iterable<DataSnapshot> children = dataSnapshot.getChildren();
+
+                        int prev_id_button = -1;
+                        int prev_id_text = -1;
+
+                        rl.removeView(questions);
```

Comment:
Same problem as before. This is an anti-pattern. Collections should be displayed using Collection Views, like RecylerView (or ListView)